addons/web/controllers/main.py

```diff
@@ -1030,10 +1030,24 @@ def test_mobile_suite(self, mod=None, **kwargs):
     def benchmarks(self, mod=None, **kwargs):
         return request.render('web.benchmark_suite')
 
+    @http.route('/web/bundle/<string:bundle_name>/<string:jscss>', type='http', methods=["get"], auth="public", cors="*")
+    def bundle(self, bundle_name, jscss):
+        js = True if "js" in jscss else False
+        css= True if "css" in jscss else False
+        all = request.env["ir.qweb"]._get_asset_nodes(bundle_name, js=js, css=css, async_load=True)
+        data = json.dumps(list(map(lambda i: {
+            "type": i[0],
+            "src": i[1]["src"],
+            "mimetype": i[1]["type"]
```

Comment:
Doesn't look like we're using the mimetype so I'm not sure we need to return it.